llvm_ir_dataset_utils/compile_time_analysis_tools/python_processing/fitting_and_plotting.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: ascii -*-
+import numpy as np
+import matplotlib.pyplot as plt
+from read_column import open_and_load
+# Usage:
+# from fitting_and_plotting import plot_functionality
```

Comment:
This should be a module level docstring.

https://google.github.io/styleguide/pyguide.html#382-modules